pyaedt/edb_core/edb_data/components_data.py

```diff
@@ -534,6 +534,21 @@ def center(self):
         center = cmpinst.GetCenter()
         return [center.X.ToDouble(), center.Y.ToDouble()]
 
+    @property
+    def bounding_box(self):
+        """Return the component bounding box.
+
+        Returns
+        -------
+        List[float]
+                [X lower left corner, Y lower left corner, X upper right corner, Y upper right corner].
+        """
+        layinst = self.edbcomponent.GetLayout().GetLayoutInstance()
+        _bbox = layinst.GetLayoutObjInstance(self.edbcomponent, None).GetBBox()
+        _pt1 = _bbox.Item1
+        _pt2 = _bbox.Item2
+        return [_pt1.X.ToDouble(), _pt1.Y.ToDouble(), _pt2.X.ToDouble(), _pt2.Y.ToDouble()]
```

Comment:
@svandenb-dev I don't think you need protected variable in this property.
```suggestion
        layer_instance = self.edbcomponent.GetLayout().GetLayoutInstance()
        bounding_box = layer_instance .GetLayoutObjInstance(self.edbcomponent, None).GetBBox()
        first_point = bounding_box.Item1
        second_point = bounding_box.Item2
        return [first_point.X.ToDouble(), first_point.Y.ToDouble(), second_point.X.ToDouble(), second_point.Y.ToDouble()]
```